fusor-ember-cli/app/routes/deployments.js

```diff
@@ -25,9 +25,13 @@ export default Ember.Route.extend(PaginationRouteMixin, {
 
   actions: {
     deleteDeployment(item) {
+      let controller = this.controllerFor('deployments');
+      let newTotalCnt = controller.get('totalCnt') - 1;
       return this.store.findRecord('deployment', item.get('id')).then(function(deployment) {
         deployment.deleteRecord();
-        deployment.save();
+        deployment.save().then(function(result) {
+          controller.set('totalCnt', newTotalCnt);
```

Comment:
Since the pagination is server side, we need to update the `totalCnt` attribute that is first populated by the API response `meta.total`, but needs to be updated after a deployment is deleted.